src/drive/web/modules/filelist/FilenameInput.jsx

```diff
@@ -1,12 +1,14 @@
 import React, { Component } from 'react'
-import Spinner from 'cozy-ui/transpiled/react/Spinner'
-import { CozyFile } from 'cozy-doctypes'
-import styles from 'drive/styles/filenameinput.styl'
+import cx from 'classnames'
 
+import { CozyFile } from 'cozy-doctypes'
```

Comment:
here it's imported from models https://github.com/cozy/cozy-drive/pull/2327/commits/541d762343b065a55f0cc5d3dcbe5f65ceea5129#diff-dc47bb533ec304621e4f8f11f3068a149cd759b50b2ea40e08186c1b51bcb045R7

Can we harmonize?

Reply:
Oui ok je vais changer partout